include/deal.II/grid/grid_tools.h

```diff
@@ -588,6 +602,17 @@ namespace GridTools
    * the unit cell, using GeometryInfo::project_to_unit_cell().  This is not
    * automatically performed by the algorithm.
    *
+   * @note When @p marked_vertices is specified the function should always be
+   * called inside a try block to catch the exception that the function might
+   * throw in the case it couldn't find an active cell surrounding the point.
+   * The motivation of using @p marked_vertices is to cut down the search space
+   * of vertices if one has apriori knowledge of collection of such vertices.
```

Comment:
-> ...if one has a priori knowledge of a collection of vertices that the point of interest may be close to.